src/js/jsx/sections/libraries/assets/CharacterStyle.jsx

```diff
@@ -88,10 +92,14 @@ define(function (require, exports, module) {
                 <div className="libraries__asset__desc">
                     <div>{font.name} {font.style}</div>
                     <div className="libraries__asset__desc-details">
-                        {fontSize.value}{fontSize.type}, {fontColorHex}
+                        {fontSize.value}{fontSize.type}
```

Comment:
This happened to me with one of my older libraries, and I couldn't get things to work again.